workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/CheckboxCard/CheckboxCard.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {ClayCheckbox} from '@clayui/form';
+
+import './CheckboxCard.scss';
+
+import classNames from 'classnames';
+
+import {Tooltip} from '../Tooltip/Tooltip';
+
+interface CheckboxProps {
+	checked: boolean;
+	description: string;
+	label: string;
+	onChange: (label: string) => void;
+	tooltip?: string;
+}
+
+export function CheckboxCard({
+	checked,
+	description,
+	label,
+	onChange,
+	tooltip,
+}: CheckboxProps) {
+	return (
+		<>
```

Comment:
You don't need this ```<></>``` here, because you already have a ```<div></div>```.

Reply:
done!